py/picca/delta_extraction/astronomical_objects/forest.py

```diff
@@ -577,21 +475,21 @@ def set_class_variables(cls, lambda_min, lambda_max,
                 np.log10(lambda_min_rest_frame) + pixel_step/2,
                 np.log10(lambda_max_rest_frame),
                 pixel_step)
-            cls.mask_fields = defaults.get("mask fields log").copy()
         elif wave_solution == "lin":
-            cls.lambda_grid = np.arange(
+            cls.log_lambda_grid = np.log10(np.arange(
                 lambda_min,
                 lambda_max + pixel_step/2,
-                pixel_step)
```

Comment:
Why is the log_lambda_grid not aligned with the input pixels? 

When running the code without rebinning (our baseline analysis in early-3d paper), this would end up shifting the wavelengths by 0.4 A, isn't it?

Reply:
it is aligned, it starts with `log(lambda_min)` as the first output pixel, and ends with the last `lambda_min+N*pixel_step` that is smaller than `lambda_max+pixel_step/2`. I guess one could make the upper end something only mildly larger than `lambda_max`, but else it's fine.

Reply:
Argh, sorry, I missread the code. I thought lambda_min had half step added. Sorry for the noise!